akita/src/plugins/filters/filter-plugin.ts

```diff
@@ -0,0 +1,14 @@
+import { AkitaPlugin, Queries } from '../plugin';
+
+
+export class FilterPlugin<E = any, S = any> extends AkitaPlugin<E, S> {
```

Comment:
Remove the file

Reply:
Do I need to have a FilterPlugins ? I did not really understood how it works  all together ? 

Because I need it for : 
`protected instantiatePlugin(id: ID): P {
    return new FilterPlugin(this.query) as P;
  }`

Or maybe juste add it, in bottom of filters-plugins.ts with no export, juste to be keep in local ? It is that ?